src/main/java/com/minecolonies/coremod/inventory/GuiMinecoloniesBuildingInventory.java

```diff
@@ -27,16 +27,16 @@ public GuiMinecoloniesBuildingInventory(final ContainerMinecoloniesBuildingInven
         this.upperChestInventory = containerMinecoloniesBuildingInventory.getPlayerInventory();
         this.lowerChestInventory = containerMinecoloniesBuildingInventory.getLowerChestInventory();
         this.allowUserInput = false;
-        int i = 222;
-        int j = 114;
+        final int i = 222;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "i" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1481)